tests/unit_tests/CMakeLists.txt

```diff
@@ -152,15 +152,15 @@ foreach(domain ${TARGET_DOMAINS})
   # Find individual tests within executable
     gtest_discover_tests(test_main_${domain}_rt
       PROPERTIES BUILD_RPATH ${CMAKE_BINARY_DIR}/lib
-      PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:$ENV{LD_LIBRARY_PATH}
+      PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:${CBLAS_LIB_DIR}:$ENV{LD_LIBRARY_PATH}
       PROPERTIES TEST_PREFIX ${DOMAIN_PREFIX}/RT/
       DISCOVERY_TIMEOUT 30
     )
   endif()
 
   gtest_discover_tests(test_main_${domain}_ct
     PROPERTIES BUILD_RPATH ${CMAKE_BINARY_DIR}/lib
-    PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:$ENV{LD_LIBRARY_PATH}
+    PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:${CBLAS_LIB_DIR}:$ENV{LD_LIBRARY_PATH}
```

Comment:
Please add `domain` check here if possible

Reply:
Addressed in the commit [f762e6f](https://github.com/oneapi-src/oneMKL/pull/251/commits/f762e6f871af3eb77133374852d1b0ccf37f97ed).

Reply:
Thank you! I have one more question regarding Windows build, we define LD_LIBRARY_PATH here for Netlib libraires but we don't define PATH on Windows, does it make sense to update PATH instead of LD_LIBRARY_PATH in case of Windows?

Reply:
I thought this is straightforward. I was wrong...

On Windows, if we try to set PATH, which is a semicolon-separated list, in `gtest_discover_tests`, for example:
``` cmake
gtest_discover_tests(test_main_${domain}_ct PROPERTIES ENVIRONMENT "PATH=C:\path1;C:\path2;C:\path3")
```
After CMake's internal processing, it turns out that the actual setting will only be `PATH=C:\path1`. This seems to be [a known CMake issue](https://gitlab.kitware.com/cmake/cmake/-/issues/21453).

Do you think we should open an issue to investigate a workaround? Right now, because users have to include Netlib in PATH to build on Windows, it's fine if they run tests directly after building. But they would encounter the "failed to load CBLAS library" error if they run tests in a separate session.

Reply:
I think it's worth adding a note to our README about Netlib required in PATH on Windows. And we also can investigate potential W/A as separated issue. I don't think we need to mix the current issue and the Windows one in this PR.

Reply:
Will be tracked in issue #252.